spec/routing/comments_routing_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'spec_helper'
 
-describe CommentsController do
+describe Supervisor::CommentsController do
```

Comment:
Could you leave the old examples and duplicate its test for the /supervisor namespace, please? The old comments controller still handles the application comments.